setup.py

```diff
@@ -18,14 +18,23 @@
 from setuptools import setup, find_packages
 import sys
 
-install_requires = ['intelhex', 'six', 'enum34', 'future', 'websocket-client', 'intervaltree', 'colorama']
+install_requires = [
+    'intelhex',
+    'six',
+    'enum34',
+    'future',
+    'websocket-client',
+    'intervaltree',
+    'colorama',
+    ]
 if sys.platform.startswith('linux'):
     install_requires.extend([
         'pyusb>=1.0.0b2',
     ])
 elif sys.platform.startswith('win'):
     install_requires.extend([
         'pywinusb>=0.4.0',
+        'hidapi',
```

Comment:
Why is `hidapi` needed on windows? Shouldn't `pywinusb` cover windows?

Reply:
It was added because there are issues with pywinusb on some boards.
Will remove and deal with separately.